pkg/apis/certmanager/v1/types.go

```diff
@@ -233,6 +233,21 @@ const (
 	UsageNetscapeSGC       KeyUsage = "netscape sgc"
 )
 
+// Keystore specific secret keys
+const (
+	// Pkcs12SecretKey is the name of the data entry in the Secret resource
+	// used to store the p12 file.
+	Pkcs12SecretKey = "keystore.p12"
+	// Data Entry Name in the Secret resource for PKCS12 containing Certificate Authority
+	Pkcs12TruststoreKey = "truststore.p12"
+
+	// JksSecretKey is the name of the data entry in the Secret resource
+	// used to store the jks file.
+	JksSecretKey = "keystore.jks"
+	// Data Entry Name in the Secret resource for JKS containing Certificate Authority
+	JksTruststoreKey = "truststore.jks"
+)
```

Comment:
These constants are already defined here:

https://github.com/cert-manager/cert-manager/blob/2db62c21c337cb5af3ea9dce1cbc0b69cfc7c509/pkg/controller/certificates/issuing/internal/keystore.go#L37-L49

I prefer where you put them, but update the existing code to also use these public constants.

Reply:
Done